applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -170,6 +172,11 @@ static void le_audio_rx_data_handler(uint8_t const *const p_data, size_t data_si
 	ret = data_fifo_pointer_first_vacant_get(&ble_fifo_rx, (void *)&iso_received, K_NO_WAIT);
 	ERR_CHK_MSG(ret, "Unable to get FIFO pointer");
 
+	if ((data_size != ENC_MAX_FRAME_SIZE) && (data_size != last_data_size)) {
```

Comment:
Should this be OR instead of AND?
Also, why is the data_size checked againts the last_data_size? It is anyway checked against the ENC_MAX_FRAME_SIZE constant.

using ENC_MAX_FRAME_SIZE is not ideal, as this is the maximum. If one changes to a different codec or config the amount of data will be less. @rick1082 can you comment?

Reply:
We should use the octets_per_sdu in here:
https://github.com/nrfconnect/sdk-nrf/blob/5f58fc4ae964e24e5b432bcfcca865f857286f62/applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c#L161
The API lc3_config_cb will be triggered when unicast server gets the configuration from unicast client.

Reply:
Added a more dynamically solution using "le_audio_config_get"